distro/configuration.nix

```diff
@@ -10,6 +10,10 @@
   imports = [ ./aster_configuration.nix ];
```

Comment:
I don't understand. The original `aster_configuration.nix` file has been removed. So how can Nix find the file referred to by this line? In addition, it seems that the original file is moved to `install_script/`. But the Git history shows that the original file is removed. You should have used `git mv` to tell Git that the file is moved, not deleted, so that Git can show the differences in `install_script/aster_configuration.nix`. Now this entire file appears to be new...